addons/gopaddle-lite/enable

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
+
+KUBE_VERSION=$(snap info microk8s | grep installed |xargs| cut -d ' ' -f 2|cut -d 'v' -f 2| cut -d '.' -f 2)
+
+CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+
+STATIC_VERSION="24"
+if [ "$KUBE_VERSION" -gt $STATIC_VERSION ]
+then 
+   echo "$KUBE_VERSION is not supported"
+fi
+ 
+
+STATICIP=""
+HELM_VERSION="4.2.3"
+STORAGECLASS=""
+ARGS=$(getopt -a --options i:v:s: --longoptions "ip:,version:,storageclass:" -- "$@")
+eval set -- "$ARGS"
+
+while true; do
+  case "$1" in
+    -i|--ip)
+      STATICIP="$2"
+      shift 2;;
+    -v|--version)
+      HELM_VERSION="$2"
+      shift 2;;
+    # -s|--storageclass)
+    #   STORAGECLASS="$2"
+    #   shift 2;;
```

Comment:
Do you plan to support storage classes as input in the future? The `microk8s enable storage` approach is only good for single node clusters as it creates local PVs.